deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -112,6 +112,8 @@ spec:
           value: /tmp/termination-log
         - name: PMEM_CSI_DRIVER_NAME
           value: pmem-csi.intel.com
+        - name: GODEBUG
+          value: x509ignoreCN=0
```

Comment:
The driver deployed via operator also requires this environment change.

Reply:
Yes. I thought I'd fire of the PR before calling it a day, but with the early consistency check it didn't get very far... Will update tomorrow...

Reply:
Fixed.